docs/releases/1.10.txt

```diff
@@ -122,12 +122,12 @@ Minor features
   to prevent an issue where Safari caches redirects and prevents a user from
   being able to log out.
 
-* Added the optional ``backend`` argument to :func:`~django.contrib.auth.login`
+* Added the optional ``backend`` parameter to :func:`~django.contrib.auth.login`
   to allow using it without credentials.
 
 * The new :setting:`LOGOUT_REDIRECT_URL` setting controls the redirect of the
   :func:`~django.contrib.auth.views.logout` view, if the view doesn't get a
-  ``next_page`` argument.
+  ``next_page`` parameter.
```

Comment:
Refers to next_page parameter
https://github.com/lachilles/django/blob/e7dc39fb65e51d7613c941f7e5768b621dea4e76/django/contrib/auth/views.py#L198-L198